xwiki-rendering-wikimodel/src/main/java/org/xwiki/rendering/wikimodel/tex/TexSerializer.java

```diff
@@ -167,9 +169,9 @@ public final void readURL(URLConnection url) throws IOException
                     System.out.println((char) in.readUnsignedByte());
                 }
             } catch (EOFException e) {
-                e.printStackTrace();
+                LOGGER.error("EOF Exception while reading url : ",e);
```

Comment:
Don't put `:`, just detail what went wrong at this level (and include the URL you failed to read). The cause exception will be printed after.

Reply:
BTW the best practices on logging is available at http://dev.xwiki.org/xwiki/bin/view/Community/JavaCodeStyle#HLoggingBestPractices